### DISCUSSION TITLE
CUR has to be created with Resource IDs

### DISCUSSION THREAD

**Discussion by milalima** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#334] [General]
After follow the documentation and enabling QuickSight I still having issues deploying the Cudos Dashboard:

>  Looking by DataSetId defined in template...complete
> 
> There are still 5 datasets missing: compute_savings_plan_eligible_spend, customer_all, ec2_running_cost, s3_view, summary_view
> Creating dataset: compute_savings_plan_eligible_spend
> Detected views:
> Missing views: compute_savings_plan_eligible_spend
> Checking if CUR is enabled and available...
>         Athena table: aws_brooklyn_dev
>         Resource IDs: no
> Error: CUR has to be created with Resource IDs

[cid.log](https://github.com/aws-samples/aws-cudos-framework-deployment/files/9376845/cid.log)

Attached is the log that is not showing any error for now:

**Comment by petrokashlikov** [comment]
This is not about Quicksight part, when you enabled Cost and Usage reports in AWS Biling console you need to select resource id's.
